src/sentry/digests/utilities.py

```diff
@@ -1,14 +1,22 @@
 from collections import Counter, OrderedDict, defaultdict
+from datetime import datetime
+from typing import Counter as CounterT
+from typing import Iterable, Mapping, MutableMapping, Optional, Set, Tuple
 
+from sentry.digests import Digest
+from sentry.eventstore.models import Event
 from sentry.models import ActorTuple, OrganizationMemberTeam, ProjectOwnership, Team, User
+from sentry.notifications.types import ActionTargetType
 
 
-# TODO(tkaemming): This should probably just be part of `build_digest`.
-def get_digest_metadata(digest):
-    start = None
-    end = None
+def get_digest_metadata(
+    digest: Digest,
+) -> Tuple[Optional[datetime], Optional[datetime], CounterT[str]]:
+    """TODO(mgaeta): This should probably just be part of `build_digest`."""
+    start: Optional[datetime] = None
+    end: Optional[datetime] = None
 
-    counts = Counter()
+    counts: CounterT[str] = Counter()
```

Comment:
is this a typo or on purpose?

Reply:
The name `Counter` is already being used by 
```python
from collections import Counter 
```
so I aliased the type:
```python
from typing import Counter as CounterT
```
I'm not totally happy with the `*T` suffix. Would `CounterType` make more sense?

Reply:
ooh yeah `CounterType` would be more clear